Makefile

```
@@ -380,7 +380,7 @@ clean-helm-rendered:
 .PHONY: clean-pprof
 clean-pprof:
 	@echo "+ $@"
-	rm /tmp/pprof.zip || true
+	rm -f /tmp/pprof.zip
```

```suggestion
	rm -f /tmp/pprof-out/pprof.zip
```
Needs verification, but I don't believe `/tmp/pprof.zip` is ever created.  The `scale-tests` target however does appear to create `/tmp/pprof-out/pprof.zip`